astropy/modeling/core.py

```diff
@@ -801,6 +807,87 @@ def inverse(self):
         raise NotImplementedError("An analytical inverse transform has not "
                                   "been implemented for this model.")
 
+    @property
+    def bounding_box(self):
+        """
+        A `tuple` of length `n_inputs` defining the bounding box limits, or 
+        `None` for no bounding box. 
+
+        The default is `None`, unless ``bounding_box_default`` is defined.
+        `bounding_box` can be set manually to an array-like  object of shape 
+        ``(model.n_inputs, 2)``. For further usage, see :ref:`bounding-boxes`
+
+        .. note:: The limits are ordered according to the `numpy` indexing 
```

Comment:
There's one 3D example in the high-level docs.
But I think it's worth mentioning that for 3D the axis order is `(z, y, x)` here.
Might be more readable if you make it a Sphinx list, like:
```
* 1D `(x_low, x_high)`
* 2D ...
* 3D ...
```